src/psyclone/tests/psyir/frontend/fparser2_where_handler_test.py

```diff
@@ -804,7 +742,9 @@ def test_where_ordering(parser):
      ("where (my_type%block(jl)%var(:) > epsi20)\n"
       "my_type%block%array(:,jl) = 3.0\n", "my_type%block(jl)%var"),
      ("where (my_type%block(jl)%var(:) > epsi20)\n"
-      "my_type%block(jl)%array(:,jl) = 3.0\n", "my_type%block(jl)%var")])
+      "my_type%block(jl)%array(:,jl) = 3.0\n", "my_type%block(jl)%var"),
+     ("where (my_type(:)%var > epsi20)\n"
+      "my_type%array(:,jl) = 3.0\n", "my_type")])
```

Comment:
For this and the next test pylint says the "fortran_writer" argument is not used.

Reply:
Removed them.